Formula/oma.rb

```diff
@@ -0,0 +1,35 @@
+class Oma < Formula
+  desc "Standalone package to infer orthologs with the OMA algorithm"
+  homepage "https://omabrowser.org/standalone/"
+  url "https://omabrowser.org/standalone/OMA.2.2.0.tgz"
+  sha256 "25ee17b92ef6631507311a93036c91154a42bcee8d033e3a3912836f2764040d"
```

Comment:
If OMA is published, please add
```ruby
# cite "http://doi.org/…"
```

Reply:
For example…
https://github.com/brewsci/homebrew-bio/blob/aec1199245fb8e47d65f2b5205d78367cb39c8e5/Formula/adam.rb#L6